packages/devtools_app/lib/src/vm_service_wrapper.dart

```diff
@@ -900,6 +878,24 @@ class VmServiceWrapper implements VmService {
       trackFuture('getRetainingPath',
           _vmService.getRetainingPath(isolateId, targetId, limit));
 
+  @override
+  Future<CpuSamples> getAllocationTraces(String isolateId,
```

Comment:
The old API calls to getAllocationSamples needs to be renamed to use getAllocationTraces and the old entry removed.  I'll create another PR to do this.

Reply:
commented on https://github.com/flutter/devtools/pull/2720/files#r579327782 - we probably need to call getAllocationSamples here for older versions of the protocol